core/templates/layout.html

```diff
@@ -24,7 +24,7 @@
 <a class="lpanel check-in" href=/checked-out data-title="Check In">Check In</a>
 <a class="lpanel">Edit Catalog</a>
 <a class="lpanel" href="https://hummedia.byu.edu/api/v2//account/login?r=https://hummedia.byu.edu">Hummedia</a>
-<a class="lpanel" data-title=Messages href="/message?">Messaging</a>
```

Comment:
I need that question mark!! haha I have changed that several times because it keeps getting deleted.